tests/utilities/test_cli.py

```diff
@@ -687,3 +690,63 @@ def __init__(self, optim1: dict, optim2: dict, scheduler: dict):
     assert isinstance(cli.model.optim1, torch.optim.Adam)
     assert isinstance(cli.model.optim2, torch.optim.SGD)
     assert isinstance(cli.model.scheduler, torch.optim.lr_scheduler.ExponentialLR)
+
+
+def test_registries(tmpdir):
+
+    assert CALLBACK_REGISTRIES.available_objects() == [
+        "BackboneFinetuning",
+        "BaseFinetuning",
+        "BasePredictionWriter",
+        "Callback",
+        "EarlyStopping",
+        "GPUStatsMonitor",
+        "GradientAccumulationScheduler",
+        "LambdaCallback",
+        "LearningRateMonitor",
+        "ModelCheckpoint",
+        "ModelPruning",
+        "ProgressBar",
+        "ProgressBarBase",
+        "QuantizationAwareTraining",
+        "StochasticWeightAveraging",
+        "Timer",
+        "XLAStatsMonitor",
+    ]
```

Comment:
This should be tested in a different file `test_registry.py`

Reply:
I don't think this is needed as it belongs to cli.